src/templates/getHandler.ts

```diff
@@ -0,0 +1,125 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+import type { NextConfig } from '../helpers/config'
+
+const { promises } = require('fs')
+const { Server } = require('http')
+const path = require('path')
+// eslint-disable-next-line node/prefer-global/url, node/prefer-global/url-search-params
+const { URLSearchParams, URL } = require('url')
+
+const { Bridge } = require('@vercel/node/dist/bridge')
+
+const { augmentFsModule, getMaxAge, getMultiValueHeaders, getNextServer } = require('./handlerUtils')
+
+const makeHandler =
+  () =>
+  // We return a function and then call `toString()` on it to serialise it as the launcher function
+  // eslint-disable-next-line max-params
+  (conf: NextConfig, app, pageRoot, staticManifest: Array<[string, string]> = [], mode = 'ssr') => {
```

Comment:
[dust] missing some return types here which makes sense and can be addressed later